src/dodal/beamlines/i22.py

```diff
@@ -345,35 +417,23 @@ def panda4(
     )
 
 
-def oav(
-    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
-) -> AravisDetector:
-    return device_instantiation(
-        NXSasOAV,
-        "oav",
-        "-DI-OAV-01:",
-        wait_for_connection,
-        fake_with_ophyd_sim,
-        drv_suffix="DET:",
-        hdf_suffix="HDF5:",
-        metadata_holder=NXSasMetadataHolder(
-            x_pixel_size=(3.45e-3, "mm"),  # Double check this figure
-            y_pixel_size=(3.45e-3, "mm"),
-            description="AVT Mako G-507B",
-            distance=(-1.0, "m"),
-        ),
-        directory_provider=get_directory_provider(),
-    )
+LINKAM_IS_IN_LAB = False
 
 
-@skip_device()
+@skip_device(
+    lambda: BL == LAB_NAME
+    and not LINKAM_IS_IN_LAB
+    or BL != LAB_NAME
+    and LINKAM_IS_IN_LAB
+)
 def linkam(
     wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
 ) -> Linkam3:
     return device_instantiation(
         Linkam3,
         "linkam",
-        "-EA-TEMPC-05",
+        "-EA-TEMPC-05" if IS_LAB else "-EA-TEMPC-05",
```

Comment:
split devices like saxs

Reply:
???I do not understand

Reply:
```python
if IS_LAB:
    return this_device()
else:
    return that_device()
```

Reply:
I do not agree that it's more readable, but I won't argue, changing now

Reply:
done

Reply:
You've done

```python
if IS_LAB:
    return this_device()
return that_device()
```

Which is functionally the same, but harder to read.

Reply:
> Which is functionally the same, but harder to read.

that's a purely stylistic opinion, usefulness of `else` is a debated point among software engineers